tests/harrisLUT/StaticTransferMatrix.csv

```diff
@@ -0,0 +1,6092 @@
+Variable Name, Static Transfer Matrix
```

Comment:
So, this would be the 3rd copy of this file! I strongly suggest you consider using the copy you are adding to ts_config_mttcs instead of adding it here and to ts_m2comm.

Reply:
I explained this problem in the PR of `ts_m2com` already. We could discuss more to have a better solution for this. Thanks!

Reply:
I think you are right. This CSC module does not need to hold the test data after the adaption of `ts_m2com`. I removed all of them in the commit: [383f261](https://github.com/lsst-ts/ts_m2/pull/30/commits/383f261f94e1b878dbe065e7e4b3506043210413). Thanks to point out this!